### ISSUE TITLE
Unexpected console log output

### ISSUE THREAD

**Issue by daniel-rikowski** [stimulus-use/stimulus-use#254]
there are two locations where some unexpected console output is happening. Not really problem, but perhaps you are still interested:

In `dist/index.js` in the next-to-last line: (only in the published npm package)

```js
console.log(`Local Stimulus Use`);
```

...which leads to log output on every page load

Also in `/src/use-window-focus/use-window-focus.ts` at line [28](https://github.com/stimulus-use/stimulus-use/blob/14e807b337ee00a48f1808e9f7185f0e1105ad49/src/use-window-focus/use-window-focus.ts#L28):

```js
console.log('i should be there')
```

**Comment by marcoroth** [comment]
Wow 🤦🏼 Thanks for catching this @daniel-rikowski. Looks like I need to be more careful with building the package locally 😅 

TL;DR: Turns out if you don't check in the changes into git and build the package locally you still end up with this debug statements in the final build. 

I'm going to release 0.51.3 momentarily!

**Comment by pySilver** [comment]
Haha. Funny I've just found out the same.

**Comment by pySilver** [comment]
Thanks for really quick update

**Comment by marcoroth** [comment]
@all-contributors add @daniel-rikowski for bug
